### ISSUE TITLE
Default color list on color plugin

### ISSUE THREAD

**Issue by AlguerGalceran** [Alex-D/Trumbowyg#454]
Hi, 

This is more a question than a bug about the colours plugin.
I'm wondering if exist any option to remove or don't show the default values, and just set the list of specific values (colours) I need.

Thanks

**Comment by Alex-D** [comment]
```
$('.editor').trumbowyg({
  plugins: {
    colors: {
      colorList: ['#fff', '#000']
    }
  }
});
```

As you can see here: https://github.com/Alex-D/Trumbowyg/blob/develop/plugins/colors/trumbowyg.colors.js#L101

And here: https://github.com/Alex-D/Trumbowyg/blob/develop/plugins/colors/trumbowyg.colors.js#L92

**Comment by AlguerGalceran** [comment]
Hi, I think I didn't explain well. 

I already did as you tell, but the colorList that I'm passing as an object doesn't replace the default values. If I'm passing two values an these are overriding the 2 first default values instead of getting rid of all the default values. See the image below.
![ex](https://cloud.githubusercontent.com/assets/20456733/21218132/4e3d1f80-c2a7-11e6-8053-2e5531b24b77.png)

The point is as I have your package installed with bower and I don't want to modify your files as if some day needs and update I will need to replace again the code on your files.

Is any option when I'm initializing "trumbowyg" to disable the default colors?

Thanks

**Comment by Alex-D** [comment]
Ah yep, it's a bug...

```js
trumbowyg.o.plugins.colors = $.extend(true, {}, defaultOptions, trumbowyg.o.plugins.colors || {});
```

should be


```js
trumbowyg.o.plugins.colors = trumbowyg.o.plugins.colors || defaultOptions;
```

Can you try it? If it works, I can fix the plugin :)

**Comment by AlguerGalceran** [comment]
Hi, tested and works perfectly.

**Comment by Alex-D** [comment]
So I will patch and release that tonight :)

**Comment by AlguerGalceran** [comment]
Thanks!

**Comment by Dannybrown2710** [comment]
@AlguerGalceran Can you please share the dropdown code that you changed for colors coming in as a grid?

**Comment by AlguerGalceran** [comment]
The plugin does it by default if you add the JS and the CSS of the plugin

**Comment by AlguerGalceran** [comment]
@Alex-D when you will be able to push that changes on the master branch? Thanks

**Comment by Alex-D** [comment]
Fix'd in 2.4.4 :)
